server/config/db-connect.js

```diff
@@ -0,0 +1,30 @@
+//db-connect.js
+/* Created by Fil on August 25, 2019 */
+
+//TODO (FG) - Change require to import once Babel works?
+const config = require('./config.js');
+const mongoose = require('mongoose');
+
+const dbUrl = `${config.db.dbType}://${config.db.host}:${config.db.port}/${config.db.name}`;
+console.log (dbUrl);
+const dbConnect = () => {
+  return mongoose.connect(dbUrl)
+};
+
+const user = new mongoose.Schema({
+  firstName: String,
+  lastName:String,
+  eMail:String
+});
+
+const User = mongoose.model('user', user);
```

Comment:
This should probably be in it's own file for a user model

Reply:
Ok, I will create a models directory and move it in there.

Reply:
I refactored the model into its own file and added unit tests for the model and db-connect files.  You can run tests by running `npm run test`